docs/docsite/rst/dev_guide/developing_modules_general_windows.rst

```diff
@@ -38,52 +38,14 @@ When creating a new module there are a few things to keep in mind:
 - Use the full cmdlet name instead of aliases, e.g. ``Remove-Item`` over ``rm``
 - Use named parameters with cmdlets, e.g. ``Remove-Item -Path C:\temp`` over ``Remove-Item C:\temp``
 
-A very basic powershell module template can be found found below:
+A very basic powershell module ``win_environment`` is included below. It demonstrates how to implement check-mode and diff-support, and also shows a warning to the user when a specific condition is met.
 
 .. code-block:: powershell
+.. include:: ../../../../lib/ansible/modules/windows/win_environment.ps1
```

Comment:
should this show the code, when I click on preview it doesn't work?

Reply:
I am not entirely sure, but Preview does not support most of the ..commands.

Reply:
I'll run make webdocs and find out :)